string alliance
string position
---
bool success
